yarn-project/end-to-end/src/e2e_client_flows/bridging.test.ts

```diff
@@ -0,0 +1,88 @@
+import { AccountWallet, PrivateFeePaymentMethod, type SimulateMethodOptions } from '@aztec/aztec.js';
```

Comment:
I'd place this file somewhere else - they're not e2e tests for client flows, but rather flows we're interested in benchmarking. We likely won't even run these tests as part of regular CI etc. A different directory inside e2e would be good I think.

Reply:
Moved to `bench` folder